aspnet/signalr/overview/performance/scaleout-with-redis.md

```diff
@@ -130,15 +130,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
 
 Add the IIS role. Include "Application Development" features, including the WebSocket Protocol.
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'WebSocket Protocol' is selected.](scaleout-with-redis/_static/image5.png)
+![Screenshot that shows a window titled Add Roles and Features Wizard On the Server Roles option WebSocket Protocol is selected.](scaleout-with-redis/_static/image5.png)
 
 Also include the Management Service (listed under "Management Tools").
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'Management Service' is selected.](scaleout-with-redis/_static/image6.png)
+![Screenshot that shows a window titled Add Roles and Features Wizard On the Server Roles option Management Service is selected.](scaleout-with-redis/_static/image6.png)
```

Comment:
How about, "Screenshot that shows Add Roles and Features Wizard. Management Service is selected."